src/docs/reference/scale-to-zero.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Scale to Zero
+---
+
+Scale to Zero means reducing unnecessary costs by ensuring you only use what you need, when you need it.
```

Comment:
I feel this sentence is way too broad and open to interpretation.

Suggestion - start with:
> App Sleeping allows you to increase the efficiency of resource utilization on Railway. It may reduce the resource usage cost of a [service](/develop/services) by ...